src/Tickets/Blocks/Ticket/app/editor/container-content/type/template.js

```diff
@@ -14,40 +14,31 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { LabeledItem } from '@moderntribe/common/elements';
-import {
-	Pass as PassIcon,
-	Ticket as TicketIcon
-} from '@moderntribe/tickets/icons';
+import { Ticket as TicketIcon } from '@moderntribe/tickets/icons';
 import './styles.pcss';
 
-const TYPES = {
-	default: {
-		title: 'Single Ticket',
-		icon: <TicketIcon />,
-	},
-
-	series: {
-		title: 'Series Pass',
-		icon: <PassIcon />,
-	},
-}
-
-const Type = ({
+const Type = ( {
 	postType,
-	seriesTitle,
 	type,
-}) => {
+	typeDescription,
+	typeIconUrl,
+	typeName,
+} ) => {
+	let ticketType = {};
 
-	const getDescription = useCallback( () => {
-		switch ( type ) {
-			case 'default':
-				return sprintf( 'A single ticket is specific to this %s.', postType );
-			case 'series':
-				return sprintf( 'A single ticket is specific to this %s. You can add a Series Pass from the %s Series page.', postType, seriesTitle );
-			default:
-				return '';
+	if ( type === 'default' ) {
+		ticketType = {
+			typeName: 'Single Ticket',
```

Comment:
Localized strings must be static:
```suggestion
			typeName: _x( 'Single Ticket', 'Default ticket type label', 'event-tickets' ),
```

Reply:
Default to using `_x` in PHP and JS code for localized strings since it allows for context.